astroplan/scheduling.py

```diff
@@ -710,10 +710,8 @@ def _get_filled_indices(self, times):
                                block in self.schedule.scheduled_blocks if
                                isinstance(block, ObservingBlock)])
         for start_end in pre_filled:
-            filled = np.where((start_end[0] < times) & (times < start_end[1]))
-            if len(filled[0]) > 0:
-                is_open_time[filled[0]] = False
-                is_open_time[min(filled[0]) - 1] = False
+            filled = np.where((start_end[0]-0.5*u.second < times) & (times < start_end[1]))
+            is_open_time[filled[0]] = False
```

Comment:
Including such a warning is probably good. Currently, I can't imagine a scenario where both `time_resolution` and `duration` are less than 1 second, which is why I opted to simply halve it. Consider revising to something like:

```
filled = np.where((start_end[0]-0.05*u.second <= times) & (times < start_end[1]))
```

Because in general I think we should mark "as filled" on one side when equal. Given that this is likely akin to floating point comparisons, subtracting a tolerance here seems logical. The addition of "<=" may provide further clarity to our intentions.